### DISCUSSION TITLE
OctoPrint

### DISCUSSION THREAD

**Discussion by Sugaryyyy** [catppuccin/catppuccin#2340] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

OctoPrint plugin for themes: https://github.com/LazeMSS/OctoPrint-UICustomizer or https://github.com/birkbjo/OctoPrint-Themeify

### Any additional comments?

OctoPrint is a Web Interface for managing 3d printers.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by 42willow** [comment]
Are you still willing to make this port? I use themeify. It might be best to fork the plugin and submit a PR directly rather than making a catppuccin port.

**Reply by Sugaryyyy** [reply]
Probably not, I've moved to Mainsail.
